ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/card.types.ts

```diff
@@ -28,6 +36,36 @@ export type ConnectionDetailsOverviewCardProps = {
   password: string;
 } & OverviewCardProps;
 
+export type AdvancedConfigurationOverviewCardProps = {
+  externalAccess: boolean;
+  parameters: boolean;
+} & OverviewCardProps;
+
+export type MonitoringConfigurationOverviewCardProps = {
+  monitoring?: string;
+} & OverviewCardProps;
+
+export type DatabaseDetailsOverviewCardProps =
+  BasicInformationOverviewCardProps &
+    ConnectionDetailsOverviewCardProps &
+    AdvancedConfigurationOverviewCardProps &
+    MonitoringConfigurationOverviewCardProps &
+    OverviewCardProps;
+
+export type ResourcesDetailsOverviewProps = {
+  numberOfNodes: DbCluster['spec']['engine']['replicas'];
```

Comment:
Is there any reason why these are not of `number` type, as they were initially?

Reply:
@dianabirs @fabio-silva this is a number. yep, but I want to suggest using the original type, so that in case of replacement, you can only change it in one place, what do you think about it?